### PR TITLE
Release procedure: fix paths for docker-build-deb.sh and docker-build-rpm.sh

### PR DESCRIPTION
**Pull request by eolivelli**
### Motivation
While preparing the release for Pulsar 2.7.2 I had a problem executing docker-build-rpm.sh and docker-build-deb.sh  on MacOS.

It looks like the reference to the Pulsar main directory is wrong and we are mounting the wrong directory while launching docker.

### Modifications

Use "realpath" in order to get the right directory.

### CODE REVIEW

**Comment by eolivelli** [comment]
the problem is present both on Linux and on Mac

this is the error on Linux for instance
```
ubuntu@ip-10-101-34-25:~/pulsar$ pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh 
centos-7: Pulling from apachepulsar/pulsar-build
75f829a71a1c: Pull complete 
7f1950a3bdcc: Pull complete 
9cfa2ce01706: Pull complete 
51b39bd58273: Pull complete 
4dbd101d2ed0: Pull complete 
201ee21c9b24: Pull complete 
5c044d9e97bd: Pull complete 
108dc2b19e5e: Pull complete 
cc981b243b9b: Pull complete 
561ee64b42eb: Pull complete 
9d3addb8f95e: Pull complete 
Digest: sha256:5c90d7a1f30f603a049cf53dc04f730c989be9bf5eeb537ec20fc874d58f74cd
Status: Downloaded newer image for apachepulsar/pulsar-build:centos-7
docker: Error response from daemon: create pulsar-client-cpp/pkg/rpm/../../../..: "pulsar-client-cpp/pkg/rpm/../../../.." includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
```

File: pulsar-client-cpp/pkg/deb/docker-build-deb.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@
 set -e
 
 ROOT_DIR=$(dirname $0)/../../..
+ROOT_DIR=$(realpath $ROOT_DIR)
```
Comment (lhotari) [inline pulsar-client-cpp/pkg/deb/docker-build-deb.sh:24]: I'd recommend replacing the 2 lines starting with "ROOT_DIR=" with this line to determine the ROOT_DIR
```
ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../.. &> /dev/null && pwd )"
```

File: pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh [hunk 2]
```diff
@@ -20,7 +20,8 @@
 
 set -e
 
-ROOT_DIR=$(dirname $0)/../../../..
+ROOT_DIR=$(dirname $0)/../../..
+ROOT_DIR=$(realpath $ROOT_DIR)
```
Comment (lhotari) [inline pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh:24]: replace with this line (same fix as made in previous comment)
```
ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../.. &> /dev/null && pwd )"
```

File: pulsar-client-cpp/pkg/deb/docker-build-deb.sh (hunk 1 above)
Comment (lhotari) [reply pulsar-client-cpp/pkg/deb/docker-build-deb.sh:24]: it looks odd, but it works and takes all possible corner cases into account. It's one most popular answer for this problem in Stackoverflow: https://stackoverflow.com/a/246128 .

Comment (eolivelli) [reply pulsar-client-cpp/pkg/deb/docker-build-deb.sh]: good. fixed. thanks
it works

File: pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh (hunk 2 above)
Comment (eolivelli) [reply pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh]: good. fixed. thanks
it works

**Comment by eolivelli** [comment]
@lhotari thanks for your quick feedback, I have updated the patch.

**Review by lhotari — APPROVED** [review]
LGTM

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**